### DISCUSSION TITLE
Completion of the Liquid Tensor Experiment

### DISCUSSION THREAD

**Discussion by giscus** [leanprover-community/blog#51] [Announcements]
# Completion of the Liquid Tensor Experiment

We are proud to announce that as of 15:46:13 (EST) on Thursday, July 14 2022 the Liquid Tensor Experiment has been
completed.
A year and a half after the
challenge
was posed by Peter Scholze we have f

http://127.0.0.1:8000/posts/lte-final/

**Comment by robertylewis** [comment]
Testing again with Eric's change
